transformations/noun_compound_paraphraser/transformation.py

```diff
@@ -0,0 +1,78 @@
+import random
+
+import os
+import json
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def noun_compound_paraphraser(text, compounds):
```

Comment:
Please add docstrings, precising the ourpose of the function, the format description of its aruments and outputs.

Reply:
Thanks for your feedback, @asnota. I will make the requested changes ASAP.